<ui version="4.0" >
 <class>preferencesDialog</class>
 <widget class="QDialog" name="preferencesDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Preferences</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="foldwatcher.qrc" >:/icons/icons/applications-system.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Reload Frequency</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="reloadSlider" >
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>99</number>
        </property>
        <property name="value" >
         <number>1</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance" >
         <bool>false</bool>
        </property>
        <property name="invertedControls" >
         <bool>false</bool>
        </property>
        <property name="tickPosition" >
         <enum>QSlider::NoTicks</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Monitor</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QCheckBox" name="animatorCheckbox" >
        <property name="text" >
         <string>Show Animation</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="okButton" >
     <property name="text" >
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cancelButton" >
     <property name="text" >
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="foldwatcher.qrc" />
 </resources>
 <connections/>
</ui>
